did/builder_test.go

```diff
@@ -0,0 +1,138 @@
+package did
+
+import (
+	"testing"
+
+	"github.com/TBD54566975/ssi-sdk/crypto"
+	"github.com/stretchr/testify/assert"
+)
+
+// Exercise all builder methods
+func TestDIDDocumentBuilder(t *testing.T) {
```

Comment:
Would it make sense to split this test into multiple smaller ones, that exercise a single behavior?

Reply:
Reasonable suggestion if we want to update that. I can put a note in for a future PR to move this into smaller tests.

Reply:
Why not do it in this PR?

From a maintainability perspective, it's easier to have smaller, self contained tests. 

FWIW, I want to add coding guidelines into the contribution doc. See https://github.com/TBD54566975/ssi-sdk/issues/290

Reply:
@andresuribe87 again, this was all using the reference of the VC Builder code, which does tests in a similar way. https://github.com/TBD54566975/ssi-sdk/blob/main/credential/builder_test.go#L254. I was being consistent with the existing codebase. I think if we want to break things out, we should also address the VC Builder as well.